src/main/java/com/launchableinc/ingest/commits/CommitGraphCollector.java

```diff
@@ -268,6 +270,10 @@ private CloseableHttpResponse handleError(URL url, CloseableHttpResponse respons
     return response;
   }
 
+  public void skipCommitMessage(boolean skipCommitMessage) {
```

Comment:
If my understanding is correct, this behavior seems strange...

When the `commitMessage` flag is true, skipCommitMessage value will be `true`. Isn't this the opposite...??


Are there any reasons that you didn't implement like the below?
```java

public void collectCommitMessage(boolean commitMessage) {
   this.collectCommitMessage
}

...

c.setMessage(collectCommit ? r.getFullMessage() : ""):
```

Reply:
I borrowed this name from the old code in mothership repository.

Reply:
If you prefer the above implementation to this current implementation, I'll modify it. Please let me know.

Reply:
I've modified based on your suggestion, and confirmed it works in my local environemnt.